deno_webgpu/shader.rs

```diff
@@ -43,6 +43,7 @@ pub fn op_webgpu_create_shader_module(
     let descriptor = wgpu_core::pipeline::ShaderModuleDescriptor {
         label: label.map(Cow::from),
         shader_bound_checks: wgpu_types::ShaderBoundChecks::default(),
+        debug: false,
```

Comment:
could an env var be added for this? Something along the lines of `DENO_WEBGPU_DEBUG_SHADERS` (and documented in the readme).

Reply:
I've done integration with instance flags and this field is removed. But I can't find how to control the flags after creating the instance. Or could I add a debug env to control shader compile?

Reply:
Yeah, an env var handled by a utility function in wgpu::util seems like the way to go.

Reply:
The expectation is that you make the decision once on instance creation - these things aren't harmful once on, and you know ahead of time if you might need debug information.

Reply:
> Yeah, an env var handled by a utility function in wgpu::util seems like the way to go.

There is a `WGPU_DEBUG` env i found which can control shader compile. What can i do to deal with deno?

Reply:
DENO is the WGPU env but with DENO on the front

Reply:
> DENO is the WGPU env but with DENO on the front

What I mean is deno test is using the `state` to get an instance
```rust
let instance = state.borrow::<super::Instance>();
```
How can I affect this by a DENO_WEBGPU_DEBUG_SHADERS env?

Reply:
given that there is `instance_flags` now and it handles multiple usecases, I'll take care of this in the coming days myself, so this isn't something this PR needs to deal with